libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTelemetryUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+class NavigationTelemetryUtils {
+  private static final String DATE_AND_TIME_PATTERN = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+  private static final SimpleDateFormat dateFormat = new SimpleDateFormat(DATE_AND_TIME_PATTERN, Locale.US);
+
+  static String obtainCurrentDate() {
```

Comment:
IMO, objects and methods should live / be where their context is. From my experience utility classes become a drawer easily and we should treat them carefully.

Reply:
@Guardiola31337 I copied this from a protected method in the telemetry library. Are you saying this should live in the event object? I imagine other navigation events will also need this method.

Reply:
Yeah, I meant where it's called / obtained. 
> I imagine other navigation events will also need this method.

For now, that's not the case so I'd defer that decision to the _Last Responsible Moment_ / to the time in which the method is called from multiple classes.

In any case, that was a minor comment that I wanted to share.